sootup.java.bytecode.frontend/src/test/java/sootup/java/bytecode/frontend/inputlocation/FixJars.java

```diff
@@ -0,0 +1,32 @@
+package sootup.java.bytecode.frontend.inputlocation;
+
+import categories.TestCategories;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import sootup.java.core.views.JavaView;
+
+@Tag(TestCategories.JAVA_8_CATEGORY)
+public class FixJars extends BaseFixJarsTest {
+
+  @Test
+  public void executeexamcorejar() {
+    String jarDownloadUrl =
+        "https://repo1.maven.org/maven2/io/github/adven27/exam-core/2024.0.10/exam-core-2024.0.10.jar";
+    String methodSignature =
+        "<io.github.adven27.concordion.extensions.exam.core.logger.LoggingFormatterExtension$LoggingFormatterListener: void afterExample(org.concordion.api.listener.ExampleEvent)>";
+    JavaView javaView = supplyJavaView(jarDownloadUrl);
+    assertMethodConversion(javaView, methodSignature);
+    assertJar(javaView);
+  }
```

Comment:
Please extract and isolate the cause of error within the jar and create a minimal test case. We do not want to download the jar each time we execute our test pipeline.